docs/DOCKER.md

```diff
@@ -34,6 +34,7 @@ POSTGRES_PASSWORD=<Non-admin user password>
 POSTGRES_PORT=<Port for postgres server Eg. 5432>
 POSTGRES_HOST=<DNS or IP address of the PostgreSQL Server, i.e. sql.somedns.org Eg. 172.25.32.1>
 LOG_LEVEL=<serilog log level i.e. Information>
+OAUTH_TOKEN_ENDPOINT=<Authentication service url>
```

Comment:
Please replace the first paragraph with this:

```none
The [Dockerfile](../src/Dockerfile) in `src` builds the Ed-Fi Data Management
Service (DMS) image directly from source code, and [nuget.Dockerfile](../src/nuget.Dockerfile)
build the image from the pre-built NuGet packages. Both versions include both the API
application and the database installer. The source code version is primarily used in local and 
end-to-end testing; the NuGet package version is intended for production-like environments
and for publishing to [Docker Hub](https://hub.docker.com/u/edfialliance).
```

Reply:
Added